### PR TITLE
feat: update numbering groups and add missing pages

### PR DESCRIPTION
**Pull request by kevinbuhmann**
- Update numbering to change 100-199 block to Foundation.
- Move color and Figma library pages to Foundation block.
- Move forms pattern page to Design Patterns block.
- Add missing pages in Foundation block based on Clarity website.
- Add links to Clarity website for documentation.

### CODE REVIEW

File: 100-color.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Using color in Clarity.
+
+## Guidance
+
+- Design teams **must** use the Clarity design tokens for colors to keep designs consistent and on-brand. See the [color documentation](https://clarity.design/documentation/color) and the [charts color documentation](https://clarity.design/documentation/charts/colors) for more information.
+
+## Changelog
+
+- **2024-12-03**: Moved to foundation and added links to website pages for documentation.
```
Comment (kevinbuhmann) [inline 100-color.md:18]:
```suggestion
- **2024-12-03**: Moved to Foundation block and added links to website pages for documentation.
```

File: 111-figma-library.md [hunk 2]
```diff
@@ -2,17 +2,18 @@
...
-Design teams consuming Clarity Components in figma.
+Consuming Clarity Components in Figma.
 
 ## Guidance
 
 - Design teams **must not** detach Clarity components in Figma. Doing so breaks alignment with the design system and makes it harder to maintain consistency.
 
 ## Changelog
 
+- **2024-12-03**: Moved to foundation.
```
Comment (kevinbuhmann) [inline 111-figma-library.md:18]:
```suggestion
- **2024-12-03**: Moved to Foundation block.
```

File: 3000-forms-pattern-design-guidance.md [hunk 3]
```diff
@@ -42,4 +42,5 @@ A [form](https://clarity.design/documentation/forms) is a collection of related
 
 ## Changelog
 
+- **2024-12-03**: Moved to design patterns section
```
Comment (kevinbuhmann) [inline 3000-forms-pattern-design-guidance.md:45]:
```suggestion
- **2024-12-03**: Moved to Design patterns block
```

File: 100-color.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+category: Foundation
+state: draft
+created: 2014-10-22
+updated: 2014-12-03
```
Comment (valentin-mladenov) [inline 100-color.md:5]:
```suggestion
created: 2024-10-22
updated: 2024-12-03
```

File: 101-grid.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+category: Foundation
+state: draft
+created: 2014-12-03
+updated: 2014-12-03
```
Comment (valentin-mladenov) [inline 101-grid.md:5]:
```suggestion
created: 2024-12-03
updated: 2024-12-03
```

File: 102-icons.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+category: Foundation
+state: draft
+created: 2014-12-03
+updated: 2014-12-03
```
Comment (valentin-mladenov) [inline 102-icons.md:5]:
```suggestion
created: 2024-12-03
updated: 2024-12-03
```

File: 103-internationalization.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+category: Foundation
+state: draft
+created: 2014-12-03
+updated: 2014-12-03
```
Comment (valentin-mladenov) [inline 103-internationalization.md:5]:
```suggestion
created: 2024-12-03
updated: 2024-12-03
```

File: 104-layouts.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+---
+category: Foundation
+state: draft
+created: 2014-12-03
+updated: 2014-12-03
```
Comment (valentin-mladenov) [inline 104-layouts.md:5]:
```suggestion
created: 2024-12-03
updated: 2024-12-03
```

**Review by valentin-mladenov — COMMENTED** [review]
Check year numbers I marked several, but there are more.

`2014` -> `2024`

**Comment by kevinbuhmann** [comment]
> Check year numbers I marked several, but there are more.
> 
> `2014` -> `2024`

Oops. Thanks! All fixed.

**Review by sandeepkulki — APPROVED** [review]

**Pull request merged**